Adds the ventriloquism behavior to disembodied heads.

Sealed101 opened the pull request:
## About The Pull Request
What is says on the tin. Same behaviour as a mime's dummy, except with more bodily injury.
The only thing this PR's missing is proper in-hands for heads but that's a much larger undertaking out of scope for this coder.
## Why It's Good For The Game
This image is begging for someone to set up a grotesque doll theatre except it's heads of your fellow crew members.
![image](https://user-images.githubusercontent.com/75863639/143828234-66b98091-bd9e-447a-9eef-7f73ac05fa10.png)

## Changelog
:cl:
expansion: Added ventriloquism to disembodied heads.
/:cl:

Sealed101 committed:
beheadening

SuperNovaa41 commented on code/modules/surgery/bodyparts/head.dm:
```diff
@@ -257,6 +257,17 @@
+/obj/item/bodypart/head/talk_into(mob/holder, message, channel, spans, datum/language/language, list/message_mods)
+	var/mob/headholder = holder
+	if(istype(headholder))
+		headholder.log_talk(message, LOG_SAY, tag="beheaded talk")
```
```suggestion
		headholder.log_talk(message, LOG_SAY, tag = "beheaded talk")
```
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#spacing

Sealed101 replied:
usingspacescanbehardsometimes

Livrah commented:
Alas, poor Yorick Terry!

Sealed101 committed:
spacing

Sealed101 committed:
prevents blood magic involvement
it is now apparent to the listener that its a head talking and not a person (so no discount voice changer(even with a robot voice))

Sealed101 committed:
prevents blood magic involved
it is now apparent to the listener that its a head talking and not a person (so no discount voice changer(even with a robot voice))

Sealed101 commented on code/modules/surgery/bodyparts/head.dm:
```diff
@@ -257,6 +257,17 @@
+	say(message, language, sanitize = FALSE)
+	return NOPASS
+
+/obj/item/bodypart/head/GetVoice()
+	return "The head of [real_name]"
+
```
this is here because otherwise you can behead a person to speak with their voice over radio, with the only drawback being the speech done in the robot text. while it's painfully obvious it's kinda not what I was going for. plus it's ventriloquism not necromancy

Sealed101 committed:
Merge branch 'advanced-ventriloquism' of https://github.com/Sealed101/tgstation into advanced-ventriloquism

MrMelbert commented:
25 bucks this breaks dullahans somehow

Sealed101 commented:
> 25 bucks this breaks dullahans somehow

funny you mentioned it, i tested it with dullahans and it seems to be working pretty much the same as it was before. i didnt test it after the latest commit but it's _probably_ fine
